content/it/agile_software_development.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Agile Software Development
+status: Completed
+category: concept
```

Comment:
Please consider translate "concept" into Italian, if needed!

Ref: "API Gateway" page in pt-br:
- https://github.com/cncf/glossary/pull/479/files
- https://deploy-preview-479--cncfglossary.netlify.app/pt-br/api_gateway/

![image](https://user-images.githubusercontent.com/46767780/155661498-18336ac5-2135-4310-ab91-6e64ea74236f.png)